src/metabase/query_processor/middleware/cache.clj

```diff
@@ -219,8 +219,7 @@
                 (save-results-xform start-time-ns metadata query-hash cache-strategy (rff metadata)))))))))
 
 (defn- is-cacheable? {:arglists '([query])} [{:keys [cache-strategy]}]
-  (and (public-settings/enable-query-caching)
-       cache-strategy))
+  cache-strategy)
```

Comment:
```suggestion
  (some? cache-strategy))
```

nit: I think this is better conveyed that it's a predicate.

Reply:
agree, done